README.md

```diff
@@ -124,6 +124,7 @@ module "my_app" {
 | tags                              | map(string)                                 | A map of AWS Tags to attach to each resource created                                                                                                                                                                                                                                                                                                                                                 | `null`                                                                                 |
 | lb_logging_enabled                | bool                                        | Option to enable logging of load balancer requests.                                                                                                                                                                                                                                                                                                                                                  | false                                                                                  |
 | lb_logging_bucket_name            | string                                      | Required if `lb_logging_enabled` is true. A bucket to store the logs in with an a [load balancer access policy](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy) attached.                                                                                                                                                              |                                                                                        |
+| lb_idle_timeout                   | number                                      | The time in seconds that the connection is allowed to be idle.                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
is there a default timeout? should we say what the AWS default timeout is here?

Reply:
Done